### ISSUE TITLE
Bidirectional gateway part 1

### ISSUE THREAD

**Issue by aspiers** [nicolasbock/github-mailing-list#10]
This is a much longer-term dream / item for the wish-list:

Somehow make it so that when people give reviews by replying to the emails this tool sends, the reviews automatically get sent back to the PR.  This could be achieved by making the tool add a magic email address to the `Cc:` header which acts as a gateway, parsing the email and submitting the review comments back via the GitHub API

This would achieve the first stage of a bidirectional gateway, i.e. that if reviews are only ever done by email, they could be read in full from the PR's web page.

See #11 for part 2.

**Comment by nicolasbock** [comment]
Yes, I totally agree. This feature would be great to have! I think the parsing part might be a little tricky though.
